py3status/modules/spaceapi.py

```diff
@@ -4,6 +4,8 @@
 Display if your favorite hackerspace is open or not.
 
 Configuration parameters:
+    button_action: Button that when clicked opens the URL sent in the space's API.
+        Setting to `0` disables. (default 3)
```

Comment:
Although other modules use `0` I think that `None` to disable sounds better but both will work in all the uses I've seen.